cate/ds/data/stores.yml

```
@@ -21,10 +21,13 @@ cci-zarr-store:
     # aws_access_key_id: ""
     # aws_secret_access_key: ""
   description: >
-    A data that contains selected datasets from the CCI Open Data Portal
-    that have been converted to [Zarr format](https://zarr.readthedocs.io/) format
+    A data store that contains selected datasets from the CCI Open Data Portal
+    which have been converted to [Zarr format](https://zarr.readthedocs.io/)
     and placed in [JASMIN](https://jasmin.ac.uk/) Object Storage.
-    This store may be successively extended.
+    Its datasets should offer a much better access performance compared to the
```

```suggestion
    Its datasets are expected to offer a much better access performance compared to the
```
Oder einfach: Its datasets offer